src/libs/protocols/protos/confidential_data_hub.proto

```diff
@@ -28,6 +28,24 @@ message SecureMountResponse {
     string mount_path = 1;
 }
 
+message ImagePullRequest {
+    // - `image_url`: The reference of the image to pull
+    string image_url = 1;
+
+    // - `bundle_path`: The path to store the bundle. After the RPC is
+    // called, Confidential Data Hub will put a `config.json` file
+    // and a mounted `rootfs` directory under the this path. Note
+    // that usually the caller and the server (CDH) runs on the same
+    // machine, so the the filesystem of the client and the server are
+    // the same. The bundle path is relative to the server's filesystem.
```

Comment:
If the client and the server are on different root filesystems (or mount namespaces), how would the client know a valid path on the server? Can we just specify that server and client are expected to run on the same root?

Reply:
If running under different ns/root, the client doesn't actually know - or needs to do it out of band. But it doesn't matter, in the context of kata-agent they are always under the same root. Let me clarify here directly that kata-agent and CDH are in the same root, so it's safe to specify a path